src/installPrompt.js

```diff
@@ -0,0 +1,24 @@
+if(window.location.search !== null && window.location.search !== "") {
+  // when plugin has a support of pwa
+  document.querySelector("html").classList.add("pwaSupported");
+}
+window.addEventListener('beforeinstallprompt', (event) => {
+  console.log('👍', 'beforeinstallprompt', event);
+  // Prevent the mini-infobar from appearing on mobile
+  event.preventDefault();
+  // Stash the event so it can be triggered later.
+  window.deferredPrompt = event;
+});
+
+// Track how the PWA was launched 
+window.addEventListener('load', () => {
+  if (navigator.fullscreen) {
+    console.log('Launched: Installed (iOS)');
+    document.querySelector("html").classList.add("pwaRunning");
+  } else if (matchMedia('(display-mode: fullscreen)').matches) {
+    console.log('Launched: Installed');
+    document.querySelector("html").classList.add("pwaRunning");
+  } else {
+    console.log('Launched: Browser Tab');
+  }
+});
```

Comment:
Codacy found an issue: [Newline required at end of file but not found.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5740126)